src/app/state/importaccounts/index.ts

```diff
@@ -3,7 +3,15 @@ import { ErrorPayload } from 'types/errors'
 import { createSlice } from 'utils/@reduxjs/toolkit'
 import { ImportAccountsListAccount, ImportAccountsState, ImportAccountsStep } from './types'
 
-export const initialState: ImportAccountsState = { accounts: [], showAccountsSelectionModal: false }
+export const initialState: ImportAccountsState = {
+  accounts: [],
+  showAccountsSelectionModal: false,
+  accountsSelectionPageNumber: 0,
+}
+
+interface ImportAccountChange extends Partial<ImportAccountsListAccount> {
```

Comment:
The original idea was that in theory, we can change any part of the account... but yes, since the only usage is to inject the balance, we can simplify it a lot more.